src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```diff
@@ -0,0 +1,21 @@
+<!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. -->
+
+<Project>
+ 
+  <UsingTask TaskName="ValidatePackage" AssemblyFile="$(DotNetPackageValidationAssembly)"/>
+  
+  <Target Name="RunPackageValidation" AfterTargets="Pack">
+    <PropertyGroup>
+      <PackagePath Condition="'$(PackagePath)' == ''">$(PackageOutputPath)$(PackageID).$(Version).nupkg</PackagePath>
+      <RunApiCompat Condition="'$(RunApiCompat)' == ''">false</RunApiCompat>
+      <ValidateWithPreviousVersion Condition="'$(ValidateWithPreviousVersion)' == ''">false</ValidateWithPreviousVersion>
```

Comment:
Avoid defining a property and use msbuild's property functions for providing defaults when passing in the property. (see suggestions below)

```suggestion
```